tests/test_prettytable.py

```diff
@@ -1,6 +1,7 @@
 import io
 import random
 import sqlite3
+from datetime import date
```

Comment:
Let's import like this:
```suggestion
import datetime as dt
```
And then use `dt.date` below.

See: https://adamj.eu/tech/2019/09/12/how-i-import-pythons-datetime-module/

Reply:
Fixed